CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## [v2.9.0](https://github.com/auth0/react-native-auth0/tree/v2.9.0) (2021-06-22)
+
+[Full Changelog](https://github.com/auth0/react-native-auth0/compare/v2.8.3...v2.9.0)
+
```

Comment:
While I understand that we are calling out the breaking change PR in this list, devs might feel that's not enough or well communicated. Should we add another big note here pointing out that this release contains a breaking change and that on [link] they can find the steps to migrate? OR, if the migration is a few paragraphs, we could add it here. Approving the PR anyways so as not to block it.

Reply:
I'll add that to the Release notes in the GH release